mne/preprocessing/tests/test_stim.py

```diff
@@ -50,6 +50,11 @@ def test_fix_stim_artifact():
     # XXX This is a very weird check...
     assert np.all(data_from_epochs_fix) == 0.
 
+    epochs = fix_stim_artifact(epochs, tmin=tmin, tmax=tmax,
+                               baseline=(-0.1, -0.05), mode='constant')
+    data = epochs.get_data()[:, :, tmin_samp:tmax_samp]
+    assert np.all(np.diff(data[0][0])) == 0.
```

Comment:
you probably also want to add an assertion something like:

```suggestion
    assert data[0][0].mean() == epochs.crop(tmin=baseline[0], tmax=baseline[1]).get_data()[0][0].mean()
```

Reply:
My code is simpler though.

Reply:
@jasmainak what happen to my pull request?

Reply:
can you add a test to check if the mean is indeed the mean of the `data` is indeed the mean of the epoch in that time interval?